Source/SG/SG.cs

```diff
@@ -24,17 +24,18 @@ public struct InstrumentConfiguration
             public double CarrierFrequency_Hz;
             public double AverageInputPower_dBm;
             public double ExternalAttenuation_dBm;
-            public string BurstStartTriggerExportTerminal;
             public bool ShareLOSGToSA;
-            public void SetDefaults()
+        }
+        public static InstrumentConfiguration GetDefaultInstrumentConfiguration()
```

Comment:
why not just make this a static method of the struct? returning self?

Reply:
Can you post a comment showing me what you're suggesting in code? You can use the following Markdown syntax with three backticks (\```) to insert a code block formatted properly for C#:
> \```c#
> // code goes here
> \```

i.e.: 
```c#
public static class MyClass {
```

Reply:
```c#
public struct InstrumentConfiguration
        {
            public string ReferenceClockSource;
            public double CarrierFrequency_Hz;
            public double AverageInputPower_dBm;
            public double ExternalAttenuation_dBm;
            public bool ShareLOSGToSA;
            public static InstrumentConfiguration GetDefault()
            {
                return new InstrumentConfiguration()
                {
                    ReferenceClockSource = RfsgFrequencyReferenceSource.PxiClock.ToString(),
                    CarrierFrequency_Hz = 1e9,
                    AverageInputPower_dBm = 0,
                    ExternalAttenuation_dBm = 0,
                    ShareLOSGToSA = true,
                };
            }
        }
```

Reply:
Oh, that's actually a really great idea. I originally had it as follows:
```c#
public struct SomeStruct {
   // Values
   public void SetDefaults()
}
```
. But based on your feedback a while ago at the .NET user meeting I changed it to the current syntax. The syntax above allowed the `SetDefaults` method to act on the created struct itself, which was nice (but required multiple lines of code in order to initialize.)

Your suggestion is ideal because it encapsulates both solutions. I'll go with that method, but I won't change it for this pull request because this affects all modules. I'll open an issue and tag you in it.